extensions/ql-vscode/src/model-editor/modeling-events.ts

```diff
@@ -25,6 +25,7 @@ interface ModeChangedEvent {
 
 interface ModeledMethodsChangedEvent {
   readonly modeledMethods: Readonly<Record<string, ModeledMethod[]>>;
+  readonly modifiedMethodSignatures: ReadonlySet<string>;
```

Comment:
If we add this to this event, we should remove `ModifiedMethodsChangedEvent`. It now seems like when modified methods are added, we fire this event, while if they are removed we fire `ModifiedMethodsChangedEvent`. This is really confusing and means you need to listen to two different events for receiving the state of 1 set.

We should then also find some other usages of the `onModifiedMethodsChanged` event which need to be updated because not all changes to this state are now sent out via that event.